fiftyone/core/state.py

```diff
@@ -113,10 +116,7 @@ class StateDescriptionWithDerivables(StateDescription):
     def __init__(self, derivables=None, *args, **kwargs):
```

Comment:
can we remove `derivables` from the `__init__()` signature then?

Reply:
Yes, missed that part. Sounds good.